2021-05-27 22:59:26,180 INFO    MainThread:19605 [wandb_setup.py:_flush():69] setting env: {}
2021-05-27 22:59:26,180 INFO    MainThread:19605 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-27 22:59:26,180 INFO    MainThread:19605 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_225926-1jwk5yn8/logs/debug.log
2021-05-27 22:59:26,180 INFO    MainThread:19605 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_225926-1jwk5yn8/logs/debug-internal.log
2021-05-27 22:59:26,180 INFO    MainThread:19605 [wandb_init.py:init():369] calling init triggers
2021-05-27 22:59:26,180 INFO    MainThread:19605 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 3, 'batch_size': 32, 'init_lr': 0.009177746681024975}
2021-05-27 22:59:26,180 INFO    MainThread:19605 [wandb_init.py:init():418] starting backend
2021-05-27 22:59:26,180 INFO    MainThread:19605 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-27 22:59:26,182 INFO    MainThread:19605 [backend.py:ensure_launched():132] starting backend process...
2021-05-27 22:59:26,221 INFO    MainThread:19605 [backend.py:ensure_launched():136] started backend process with pid: 21917
2021-05-27 22:59:26,223 INFO    MainThread:19605 [wandb_init.py:init():423] backend started and connected
2021-05-27 22:59:26,224 INFO    MainThread:19605 [wandb_init.py:init():465] updated telemetry
2021-05-27 22:59:26,225 INFO    MainThread:19605 [wandb_init.py:init():484] communicating current version
2021-05-27 22:59:27,013 INFO    MainThread:19605 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-27 22:59:27,013 INFO    MainThread:19605 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-27 22:59:27,322 INFO    MainThread:19605 [wandb_init.py:init():522] starting run threads in backend
2021-05-27 22:59:27,360 INFO    MainThread:19605 [wandb_run.py:_console_start():1565] atexit reg
2021-05-27 22:59:27,361 INFO    MainThread:19605 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-27 22:59:27,361 INFO    MainThread:19605 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-27 22:59:27,363 INFO    MainThread:19605 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-27 22:59:27,363 INFO    MainThread:19605 [wandb_init.py:init():546] run started, returning control to user process
2021-05-27 23:03:12,349 INFO    MainThread:19605 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 255
2021-05-27 23:03:12,349 INFO    MainThread:19605 [wandb_run.py:_restore():1507] restore
2021-05-27 23:03:14,356 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:03:14,460 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:03:14,674 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:03:15,428 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 3808
}

2021-05-27 23:03:15,836 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3810
  total_bytes: 3810
}

2021-05-27 23:03:15,938 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3810
  total_bytes: 3810
}

2021-05-27 23:03:16,041 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3810
  total_bytes: 3810
}

2021-05-27 23:03:16,142 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3810
  total_bytes: 3810
}

2021-05-27 23:03:16,245 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3810
  total_bytes: 3810
}

2021-05-27 23:03:16,347 INFO    MainThread:19605 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3810
  total_bytes: 3810
}

2021-05-27 23:03:17,436 INFO    MainThread:19605 [wandb_run.py:_show_files():1879] logging synced files
